services/app/src/components/gameFooter/game-footer.css

```diff
@@ -1,10 +1,10 @@
 /* -----Game Footer----- */
 #game-footer {
-  box-sizing: border-box;
   display: flex;
   flex-direction: column;
-  width: 55%;
-  margin: 0 auto 10px auto;
+  width: 540px;
+  padding: 16px 16px 8px 8px;
```

Comment:
use spacing classes

Reply:
Is it really necessary in this case? We would change this line with 4 separate classes?

Reply:
It is necessary to avoid having multiple places that define spacing. That is not maintainable.

I just not realised that you have top and right padding set to 16px while bottom and left padding is set to 8px. That is very unusual thing to see, why do you need that?